Update defender-for-containers-vulnerability-assessment-azure.md (#102603)
Azure/azure-content: articles/defender-for-cloud/defender-for-containers-vulnerability-assessment-azure.md

Some images may reuse tags from an image that was already scanned. For example, you may reassign the tag “Latest” every time you add an image to a digest. In such cases, the ‘old’ image does still exist in the registry and may still be pulled by its digest. If the image has security findings and is pulled, it will expose security vulnerabilities.

> ### Does Defender for Containers scan an images in Microsoft Container Registry?

Reviewer: ```suggestion
### Does Defender for Containers scan images in Microsoft Container Registry?
```

> Currently Defender for Containers can scan Images in Azure Container Registry(ACR) and AWS Elastic Container Registry (ECR) only

Reviewer 2: ```suggestion
Currently, Defender for Containers can scan images in Azure Container Registry (ACR) and AWS Elastic Container Registry (ECR) only.
```

> Docker Registry, Microsoft Artifact Registry / Microsoft Container Registry or ARO built-in container image registry are not supported.

Reviewer 2: ```suggestion
Docker Registry, Microsoft Artifact Registry/Microsoft Container Registry, and Microsoft Azure Red Hat OpenShift (ARO) built-in container image registry are not supported.
```

> Images should first be imported to ACR. To know more about importing container images to an Azure container registry: https://learn.microsoft.com/en-us/azure/container-registry/container-registry-import-images?tabs=azure-cli

Reviewer 2: ```suggestion
Images should first be imported to ACR. Learn more about [importing container images to an Azure container registry](/azure/container-registry/container-registry-import-images?tabs=azure-cli)
```

> Images should first be imported to ACR. Learn more about [importing container images to an Azure container registry](/azure/container-registry/container-registry-import-images?tabs=azure-cli)

Reviewer: ```suggestion
Images should first be imported to ACR. Learn more about [importing container images to an Azure container registry](/azure/container-registry/container-registry-import-images?tabs=azure-cli).
```
